config/manager/SonataFlow-Builder.containerfile

```diff
@@ -16,7 +16,7 @@ RUN /home/kogito/launch/build-app.sh ./resources
 #=============================
 # Runtime Run
 #=============================
-FROM registry.access.redhat.com/ubi9/openjdk-17:latest
+FROM registry.access.redhat.com/ubi9/openjdk-17-runtime:latest
```

Comment:
regarding my previous comment, here we have ubi9

Reply:
Ops, that must be changed.